h/h_api/resources/schema/bulk_api/wrapper.json

```diff
@@ -9,11 +9,13 @@
         {"enum":  ["create", "upsert", "configure"]},
         {"type":  "object"}
     ],
+    "minItems": 2,
+    "maxItems": 2,
+
     "if": {
         "items": [
             {"enum": ["configure"]}
-        ],
-        "additionalItems": true
```

Comment:
This was not setting the minimum. I thought the "tuple" way of specifying schema specified the length, but it just sets the contents of the positions _if they are filled_.